src/cmd/common.go

```diff
@@ -34,12 +34,13 @@ const (
 	architecture   configOption = "architecture"
 	noLogFile      configOption = "no_log_file"
 	noProgress     configOption = "no_progress"
+	ociConcurrency configOption = "oci_concurrency"
 )
```

Comment:
Looks like there is another config option missing too: `no_color`

Reply:
nice catch, thank you!